android/sdk/src/main/java/com/taobao/weex/layout/measurefunc/TextContentBoxMeasurement.java

```diff
@@ -335,9 +336,10 @@ private float getTextWidth(TextPaint textPaint, float outerWidth, boolean forceT
   @NonNull
   Layout createLayout(final float textWidth, @Nullable Layout previousLayout) {
     Layout layout;
+    boolean forceRtl = mComponent.isNativeLayoutRTL();
     if (previousWidth != textWidth || previousLayout == null) {
-      layout = new StaticLayout(spanned, mTextPaint, (int) Math.ceil(textWidth),
-              Layout.Alignment.ALIGN_NORMAL, 1, 0, false);
+      layout = StaticLayoutProxy.create(spanned, mTextPaint, (int) Math.ceil(textWidth),
```

Comment:
There is some problem for `StaticLayoutProxy.create` :

*  `StaticLayoutProxy.create` is slow due to reflection
*  reflection is not reliable on tremendous Chinese ROMs.